src/main/java/entity/Task.java

```diff
@@ -30,27 +35,33 @@ public enum Priority {
      * @param dates      Time window of the task
      * @param priority   Task priority (defaults to MEDIUM if null)
      */
-    public Task(Info info, BeginAndDueDates dates, Priority priority) {
+    public Task(Info info, BeginAndDueDates dates, Priority priority, Boolean regular) {
```

Comment:
```suggestion
    public Task(Info info, BeginAndDueDates dates, Priority priority) {
```
The ```regular``` parameter represents application logic, not core domain data. Business rules about task regularity should be handled in Use Cases, not Entity constructors.